test/metabase/cmd/dump_to_h2_test.clj

```diff
@@ -1,11 +1,21 @@
 (ns metabase.cmd.dump-to-h2-test
   (:require [clojure.java.io :as io]
+            [clojure.java.jdbc :as jdbc]
             [clojure.test :refer :all]
-            [metabase.cmd
-             [copy :as copy]
-             [dump-to-h2 :as dump-to-h2]]
             [metabase.cmd :as cmd]
-            [metabase.util.files :as u.files]))
+            [metabase.cmd.copy :as copy]
```

Comment:
If the version of `master` you branched off of was using the nested `:require` style, it must be a little out of date -- we swtiched over 2 weeks ago I think. Maybe try merging `master` in to this PR or even rebasing the branch on `origin/master` and seeing if that gets CodeCov to stop being weird

Reply:
I checked and master has the nested version. maybe when the refactor was done, this file was not in the repo?

Anyway, I rebased to master just in case, but probably this diff will still appear